leasing/models/rent.py

```diff
@@ -844,6 +921,14 @@ def is_active_on_period(self, date_range_start, date_range_end):
 
         return False
 
+    def set_start_price_index_point_figure(self):
+        if self.old_dwellings_in_housing_companies_price_index:
+            start_index_number_yearly = IndexNumberYearly.objects.get(
+                index=self.old_dwellings_in_housing_companies_price_index,
+                year=self.start_date.year - 1,
```

Comment:
Should this be the Lease's start date, instead of Rent's start date? Based on the ticket comment.

Reply:
Fixed in a later commit